logstash-output-http.gemspec

```
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name            = 'logstash-output-http'
-  s.version         = '5.2.4'
+  s.version         = '5.2.5'
```

This is a new feature so it deserve a minor version. Actually `main` has `5.5.0` so the target could be `5.6.0`

```suggestion
  s.version         = '5.6.0'
```